datarobot_provider/operators/datarobot.py

```diff
@@ -249,13 +249,36 @@ def execute(self, context: Dict[str, Any]) -> str:
         # Initialize DataRobot client
         DataRobotHook(datarobot_conn_id=self.datarobot_conn_id).run()
 
+        score_settings = context["params"]["score_settings"]
+        intake_settings = score_settings["intake_settings"]
+        intake_type = intake_settings["type"]
+
         # Score data
         self.log.info(
             f"Scoring predictions against deployment_id={self.deployment_id} "
-            f"with settings: {context['params']['score_settings']}"
+            f"with settings: {score_settings}"
         )
+
+        # The BatchPredictionJob API expects an instance of a dataset, not just
```

Comment:
That's not accurate. The [API expects](https://app.datarobot.com/apidocs/autodoc/api_reference.html#post--api-v2-batchPredictions-) `datasetId`, but the [python API client expects](https://datarobot-public-api-client.readthedocs-hosted.com/en/v3.0.1/autodoc/api_reference.html#datarobot.models.BatchPredictionJob.score) `dataset` object.